chain/chain/src/chain.rs

```diff
@@ -2533,7 +2533,7 @@ impl<'a> ChainUpdate<'a> {
             return Err(ErrorKind::Orphan.into());
         }
 
-        if block.header.inner_lite.height > head.height + self.epoch_length {
+        if block.header.inner_lite.height > head.height + self.epoch_length * 2 {
```

Comment:
Do we still need this kind of check with doomslug and NFG? It seems extremely unlikely that someone would be able to produce a block that far into the future.

Reply:
It doesn't hurt. It would only surface itself in some very bad cases, but doesn't hurt much.
If cross_shard_tx continues hitting it, we can revisit and see if we can convince ourselves it can be removed.